src/redteam/util.ts

```diff
@@ -68,9 +138,6 @@ export function isEmptyResponse(response: string): boolean {
 
 export function isBasicRefusal(response: string): boolean {
   const lowerResponse = normalizeApostrophes(response.trim().toLowerCase());
-  if (lowerResponse.includes('\n')) {
-    return false;
-  }
```

Comment:
slightly nervous about this one but I will trust you evaled it

Reply:
Anthropic in particular returns multi-paragraph refusals